frontend/plan/src/constants.js

```diff
@@ -0,0 +1 @@
+export const BACKEND_URL = process.env.REACT_APP_BACKEND_URL;
```

Comment:
Can this be `BACKEND_DOMAIN`? Just since we're following that notation elsewhere?

Reply:
I don't think DOMAIN is accurate in this case; for example, in prod, the value is `https://api.penncourses.org/plan`.

Reply:
Oh I see, I didn't realize you were scoping the the `plan` path. That makes sense.